src/app/components/PaymentSummary.tsx

```diff
@@ -8,7 +8,7 @@ function PaymentSummary({ amount, amountAlt, description }: Props) {
   return (
     <dl className="mb-0">
       <dt className="font-medium text-gray-800 dark:text-white">Amount</dt>
-      <dd className="mb-0 text-gray-600 dark:text-gray-500">{amount}</dd>
+      <dd className="mb-0 text-gray-600 dark:text-gray-500">{amount} sat</dd>
```

Comment:
We use sat or sats?

Reply:
twitter said: https://twitter.com/getAlby/status/1509884716517928963

Reply:
https://github.com/getAlby/lightning-browser-extension/issues/308